include/igl/opengl/glfw/Viewer.h

```diff
@@ -72,7 +72,8 @@ namespace glfw
     IGL_INLINE bool save_scene(std::string fname);
     // Draw everything
     IGL_INLINE void draw();
-    // OpenGL context resize
+    // OpenGL context get size / resize
+    IGL_INLINE void get_size(int &w, int &h); // helper func for getting the size of current window
```

Comment:
The function should return a `std::pair<int, int>` rather than use output args.

Reply:
is `libigl` c++17 compatible?

Reply:
so that `return {w, h}` is acceptable for `std::pair<int, int>`?

Reply:
code updated. Please let me know if it needs additional edit.

Reply:
Returning `{w, h}` is just C++11, building a std::pair from an initializer list. If you were to retrieve those values using a structured binding `auto [w, h]` then it would be C++17.

Reply:
Yes, I use that in my code. OK, then all good. Thank you for the explanation.